modules/serverless-logic-upgrading-osl-operator.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-upgrading-operator-from-1.34-to-1.35
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-logic-upgrading-osl-operator_{context}"]
+= Upgrading the {ServerlessLogicOperatorName}
+
+To transition from OSL version 1.34.0 to 1.35.0, you must upgrade the {ServerlessLogicOperatorName} using the OpenShift web console. This upgrade ensures compatibility with newer features and proper functioning of your workflows.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have {ServerlessLogicOperatorName} installed on your cluster.
+* You have access to a {ServerlessLogicProductName} project with the appropriate roles and permissions to create applications and other workloads in {ocp-product-title}.
+* You have access to the *OpenShift Management Console* for Operator upgrades.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. In the web console, navigate to the *Operators* -> *OperatorHub* -> *Installed Operator* page.
+. Select the `openshift-serverless-logic` namespace from the *Installed Namespace*.
+. In the list of installed operators, find and click the {ServerlessLogicOperatorName}.
+. In the Operator details page, click on the *Subscription* tab, Click *Edit Subscription*.
+. In the *Update* channel, select, `stable-1.35` and click *Save* to apply the changes.
+. To monitor the upgrade process, execute the following command: 
++
+[source,terminal]
+----
+$ oc get subscription serverless-logic-operator -n openshift-serverless-logic -o jsonpath='{.status.currentCSV}'
```

Comment:
```suggestion
oc get subscription logic-operator-rhel8 -n openshift-serverless-logic -o jsonpath='{.status.installedCSV}'
```

Reply:
Updated! I am keeping the *$* sign as per the style guide. If we mention any command, we must put the *$* before it.